src/Api/Reporting/Background/ActiveUserReportingBackgroundService.cs

```diff
@@ -0,0 +1,39 @@
+using System.Data;
+using Passwordless.Common.Background;
+using Passwordless.Service;
+
+namespace Passwordless.Api.Reporting.Background;
+
+public sealed class ActiveUserReportingBackgroundService : BasePeriodicBackgroundService
+{
+    private readonly IServiceProvider _serviceProvider;
+
+    public ActiveUserReportingBackgroundService(
+        IServiceProvider serviceProvider,
+        TimeProvider timeProvider,
+        ILogger<ActiveUserReportingBackgroundService> logger) : base(
+        new TimeSpan(22, 0, 0),
```

Comment:
Nit: I wish this was indented one more level since its part of the base class and not the current class.